scripts/isolate_tests.py

```diff
@@ -43,7 +43,7 @@ def extract_docs_cases(path):
     tests = []
 
     # Collect all snippets of indented blocks
-    for l in open(path, mode='r', encoding='utf8').read().splitlines():
+    for l in open(path, mode='r', errors='ignore', encoding='utf8').read().splitlines():
```

Comment:
Should this be its own PR?

Reply:
I can pull it out. I just stumbled over that while testing and quickly fixed it